packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -865,6 +865,7 @@ it('should filter table', async () => {
   expect(onFilter).toHaveBeenCalledWith(
     [{ fieldType: 'text', filterType: 'text', id: 'name', value: '2' }],
     expect.objectContaining({ filters: [{ id: 'name', value: '2' }] }),
+    expect.any(Object),
```

Comment:
Does not seem like a valid test. At least check returned rows count to see if it match.